web-ui/src/components/feedback_template_selector/FeedbackTemplateSelector.jsx

```diff
@@ -81,35 +135,36 @@ const FeedbackTemplateSelector = ({changeQuery}) => {
       <TemplatePreviewModal
         template={preview.selectedTemplate}
         open={preview.open}
-        onSubmit={() => handlePreviewSubmit(preview.selectedTemplate)}
+        onSubmit={(submittedTemplate) => handlePreviewSubmit(submittedTemplate)}
         onClose={() => handlePreviewClose(preview.selectedTemplate)}
       />
       }
+      <div className="feedback-ad-hoc-action-buttons">
+        <Button
+          className="ad-hoc-button"
+          variant="contained"
+          color="primary"
+          onClick={onNewAdHocClick}>
+          New Ad-Hoc Template
+        </Button>
+        <Tooltip title="An ad-hoc template allows you to ask a single question" arrow>
+          <HelpOutlineIcon style={{color: "gray", marginLeft: "10px"}}/>
+        </Tooltip>
+      </div>
       <div className="card-container">
```

Comment:
Whenever I load up the feedback page it throws a TypeError saying that templates is undefined in FeedbackTemplateSelector.jsx.

Reply:
I could not figure out how to reproduce this error, please let me know if it's still happening after the most recent changes.

Reply:
@zsbrown97 are you still seeing the error ?